### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Shazeb314**

### CODE REVIEW

File: views/auth/register.ejs [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
\ No newline at end of file
+  <div class="container">
+    <div class = "img-store">
+     <img src = "books.png" alt = "Sign In">
+    </div>
+    <div class="content">
+       <a href="Sign In/index.html"><i class = "bi-arrow-left"></i></a> 
+        <h1>Sign Up</h1>
+        <form action = "/auth/register" method = "POST">
+            <input type = "text" placeholder = "Username" class = "details" required >
```
[the remark this answers is not in the document]
Comment (Lalit3716) [reply views/auth/register.ejs:12]: Yeah try to give them on your own see login.ejs for reference.
I will fix them later if you gave wrong attributes

[the remark this answers is not in the document]
Comment (Shazeb314) [reply views/auth/register.ejs:12]: Ok

File: views/auth/register.ejs [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+            <br>
+            <input type = "text" placeholder = "Email Address" class = "details" required >
+            <br>
+            <input type = "password" placeholder = "Create Password" class = "details" required>
+            <br>
+            <input type = "text" placeholder = "Confirm Password" class = "details" required>
+            <br>
+            <h5>Fields of Interest</h3>
+            <input type = "checkbox"><label>&nbspComputer Science</label>
```
[the remark this answers is not in the document]
Comment (Shazeb314) [reply views/auth/register.ejs:21]: I didnt understand this completely can you elaborate

[the remark this answers is not in the document]
Comment (Shazeb314) [reply views/auth/register.ejs:21]: I mean should i give name = foi[] as it is?

[the remark this answers is not in the document]
Comment (Lalit3716) [reply views/auth/register.ejs:21]:
```
<input type = "checkbox" name="foi[]">
```
You have to do this for every checkbox Input element for backend.

[the remark this answers is not in the document]
Comment (Lalit3716) [reply views/auth/register.ejs:21]: > I mean should i give name = foi[] as it is?

Yes

[the remark this answers is not in the document]
Comment (Shazeb314) [reply views/auth/register.ejs:21]: ok

**Pull request merged**